documentation/docs/examples/how-to-run-examples.mdx

```diff
@@ -42,8 +42,8 @@ the following `.env` file:
 ```bash
 MONGO_URL=mongodb://admin:admin@localhost:27017
```

Comment:
sometimes we have admin admin and sometimes root + rootPassword

can you please replace all references to root:rootPassword

in addition here it should not be localhost but 0.0.0.0 so it would look as: 
```MONGO_URL=mongodb://root:rootpassword@0.0.0.0:27017```

Reply:
also for the other references

Reply:
Updated